src/.babelrc

```diff
@@ -1,12 +1,9 @@
 {
     "presets": [
-      ["latest", {
-        "es2015": {
-            "modules": false
-        }
-      }]
+        ["@babel/preset-env", { "modules": false }]
```

Comment:
Looks like if no targets or browserslist is specified, then we'll get transpilation of all es2015+ code by default (except modules which is explicitly excluded).

This is definitely beneficial so we're not asking our consumers to transpile our code.  But, I wonder if we even need the present-env in this case since we won't be using it to target browsers with support?

Finally, rollup is recommending that libs not distribute non-transpiled code.  When I originally did the bundling, I took a different approach in that I expected an esm consumer to need to transpile.  But, with the hope of someday using esm directly, maybe this concept has changed.

Let's chat on this a bit tomorrow as well.

Reply:
So, I think we still need preset-env as without it babel won't transpile our code at all. I believe that babel's default behavior, with no presets/plugins, just leaves the code untouched. It seems that just using this preset, while it is heavy-handed when we don't have a browser list and by default compiling all es2015+ down to es5, is pretty reasonable. Perhaps we should consider what browsers to target that way we could be a bit more efficient here and compile less if we don't have to. I'm not sure what browsers we target as a company, but maybe we can get that list and just use that for the SDK as well. Let me know your thoughts on this.